docs/book/src/clusterctl/configuration.md

```diff
@@ -160,9 +160,16 @@ all the images in the cert-manager component.
 For situations when resources are limited or the network is slow, the cert-manager wait time to be running can be customized by adding a field to the clusterctl config file, for example:
 
 ```yaml
-  cert-manager-timeout: 15m 
+  cert-manager-timeout: 15m
 ```
 
 The value string is a possibly signed sequence of decimal numbers, each with optional fraction and a unit suffix, such as "300ms", "-1.5h" or "2h45m". Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
 
-If no value is specified or the format is invalid, the default value of 10 minutes will be used.
\ No newline at end of file
+If no value is specified or the format is invalid, the default value of 10 minutes will be used.
+
+
+## Debugging/Logging
+
+To have more verbose logs you can use the `-v` flag when running the `clusterctl` and set the level of the logging verbose from 0 to 5.
```

Comment:
Nit: I'd indicate that 5 is the highest verbosity level, but maybe that doesn't matter as it is standard with kubectl and other kubernetes tools.

Reply:
On second thought, I'd just leave out the levels of verbosity. I've seen  `kubectl` with `-v9` produce fewer logs than `-v99` recently. Unless the CAPI community has already decided on log levels I don't know if we should mention it here. Because once it's documented it will be hard to change for whatever reason.

Reply:
Heh, it is sadly complicated. The first task I attempted to help with in Kubernetes was to make changes to a couple components to line up with the Logging Convention published by SIG Instrumentation (which is 0-5) but got pushback on the commits as some components don't want to change.  
klog does allow any integer value as a log level, but a quick grep through the cluster-api code will show that the only two values used are `log.V(1)` and `log.V(5)`, so the range is valid if we want to document it.

Reply:
I will remove and we can set back in a followup if needed